components/desktop/partials/KnitProjectsWidgetItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="knit-projects-widget-item">
+    <div class="knit-projects-widget-item__top-section">
+      <img :src="projectLogoSource || 'images/teal-star.png'" alt="" class="knit-projects-widget-item__project-logo">
+      <h3 class="knit-projects-widget-item__title"> {{ title }} </h3>
+    </div>
+    <div class="knit-projects-widget-item__bottom-section">
+      <ul class="knit-projects-widget-item__list">
+        <li v-if="isNotEmpty(tags)" class="knit-projects-widget-item__list-item">
+          <span class="knit-projects-widget-item__list-item-logo flaticon-curly-brackets"/>
+            {{ arrayToStringList(tags) }}
+        </li>
+        <li v-if="maxTeamSize" class="knit-projects-widget-item__list-item">
+          <span class="knit-projects-widget-item__list-item-logo flaticon-user"/>
+            Max: {{ maxTeamSize }}, Aktualnie: {{ !currentTeamSize ? 0 : currentTeamSize }}
+        </li>
+        <li v-if="isNotEmpty(repoLink)" class="knit-projects-widget-item__list-item">
+          <span class="knit-projects-widget-item__list-item-logo flaticon-github-logo"/>
+            <a :href="repoLink" class="knit-projects-widget-item__list-item-link">
+              {{ trimString(repoLink, 50) }}
+            </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+import templateHelper from '~/helpers/templateHelper.js'
+
+export default {
+  data () {
+    return {}
+  },
+  props: [
+    'tags',
+    'title',
+    'repoLink',
+    'maxTeamSize',
+    'currentTeamSize',
+    'projectLogoSource'
+  ],
+  computed: {},
+  mixins: [ templateHelper ]
+}
+</script>
+
+<style lang="scss">
+@import "assets/scss/imports.scss";
+.knit-projects-widget-item {
+  border-top: 1px solid $knit-projects-widget-item-border-color;
+  border-bottom: 1px solid $knit-projects-widget-item-border-color;
```

Comment:
You have double borders. Use only border-bottom with `:last-child` no bordered ('see-more' button should have a top border). Another trick is to use `margin-top: -1px` in every project item.

Reply:
@zrebcu411 I prefer another trick thanks :)